pmaweb/templates/sponsors.html

```diff
@@ -98,24 +98,24 @@ <h2 id="partners">Technology partners</h2>
 <p>
 The phpMyAdmin project is grateful to our technology partners for their services.</p>
 
-<h3><a href="https://www.cdn77.com/"><img src="{% static 'images/cdn77_logo.png' %}" alt="CDN77.com" /></a></h3>
+<h3><a rel="nofollow" href="https://www.cdn77.com/"><img src="{% static 'images/cdn77_logo.png' %}" alt="CDN77.com" /></a></h3>
 
-<p>Our downloads are accelerated by <a href="https://www.cdn77.com/">content delivery network</a>.</p>
+<p>Our downloads are accelerated by <a rel="nofollow" href="https://www.cdn77.com/">content delivery network</a>.</p>
 
-<h3><a href="https://github.com/phpmyadmin"><img src="{% static 'images/github.png' %}" alt="GitHub" /></a></h3>
+<h3><a rel="nofollow" href="https://github.com/phpmyadmin"><img src="{% static 'images/github.png' %}" alt="GitHub" /></a></h3>
 
 <p>GitHub provides hosting for our source code and issue tracker.</p>
 
-<h3><a href="https://browserstack.com"><img src="{% static 'images/BrowserStackLogo.png' %}" alt="BrowserStack" /></a></h3>
+<h3><a rel="nofollow" href="https://browserstack.com"><img src="{% static 'images/BrowserStackLogo.png' %}" alt="BrowserStack" /></a></h3>
 
 <p>BrowserStack is running our Selenium tests in real browsers.</p>
 
-<h3><a href="https://scrutinizer-ci.com/g/phpmyadmin/phpmyadmin/"><img src="{% static 'images/scrutinizer.png' %}" alt="Scrutinizer" /></a></h3>
+<h3><a rel="nofollow" href="https://scrutinizer-ci.com/g/phpmyadmin/phpmyadmin/"><img src="{% static 'images/scrutinizer.png' %}" alt="Scrutinizer" /></a></h3>
 
 <p>Scrutinizer helps us improve code quality.</p>
 
-<h3><a href="https://weblate.org/"><img src="{% static 'images/weblate-button.png' %}" alt="Weblate" /></a></h3>
+<h3><a rel="sponsored" href="https://weblate.org/"><img src="{% static 'images/weblate-button.png' %}" alt="Weblate" /></a></h3>
```

Comment:
I think all technology partners should have same treatment, I don't see why Weblate should be different here. All of these are providing some services to the project for free. I have no clue which of rel is better fit here.

Reply:
I used sponsored, seems to fit the best